low-level/sspl-ll.spec

```diff
@@ -94,6 +94,7 @@ chmod 644 $STATE_FILE
 mkdir -p /var/%{product_family}/sspl/bundle /var/log/%{product_family}/sspl /etc/sspl
 SSPL_DIR=/opt/seagate/%{product_family}/sspl
 CFG_DIR=$SSPL_DIR/conf
+rm -f /var/cortx/sspl/data/server/SAS_PORT_SENSOR_DATA*
```

Comment:
@mohitpathak1 rm should ideally go to `%postun`

Reply:
so %post would remove any pre-existing file on pkg installation and %postun would remove file on current pkg uninstallation, and agree latter is cleaner as it wont keep any unwanted files on system. 
However for update case, for older rpms where this change(%postun step)  in not in place, wont execute this cleanup and we would end up hitting same issue, so safer in %post stage only.

@ypise ?

Reply:
Right `Update = Uninstall + Install`
Ref: https://www.golinuxhub.com/2018/05/how-to-execute-script-at-pre-post-preun-postun-spec-file-rpm/

Reply:
Moved the change to `%postun`

Reply:
>In post stage, the file is being removed unchecked, so if file is missing, installation would fail.
we have "rm -f" to avoid failing that step, if file exists or not.

Secondly for update case, lets say build 128 -> build 146 failed, now with build 147 we will add this rm -f step under %postun stage of rpm spec file, which would get executed from build 147 rpm uninstallation & stale file would be cleaned up. so all updates from build 147 to any forward build will not have this issue.

But what about update from builds previous to 147, like 128 (or till 146) to 147 (or further build) update, it would still fail, given there is no %postun step which would executed on build <=146, so stale file will remain as it is. To avoid this situation, if we cleanup in %post stage from build 147 onwards, update would pass from any build <147 to build >= 147, given stale file would get cleaned on new rpm installation.

If not clear, lets have a quick call. Ping me when you are available.

Reply:
After discussion with Surabh, realize we need this cleanup stage at %postun in case of rollback too (like 147 fails to install, then it should clean up this file). And in %post, for reason what i have mentioned above